src/SonarQube.Client/Models/SonarQubeIssue.cs

```diff
@@ -59,7 +59,7 @@ public class SonarQubeIssue
         public string Message { get; }
         public string ModuleKey { get; }
         public string RuleId { get; }
-        public bool IsResolved { get; }
+        public bool IsResolved { get; set; }
```

Comment:
Probably worth commenting that this property needs to be mutable since it can be changed later by  SLVS.